templates/news/news-en.html

```diff
@@ -3,224 +3,448 @@
 	"title": "News",
 	"language": "en",
 	"altLangPage": "news-fr.html",
-	"breadcrumbs": false,
 	"secondlevel": false,
-	"dateModified": "2023-09-01",
+	"dateModified": "2024-09-13",
 	"share": "true",
 	"pageType": "news"
 }
 ---
-<div class="mrgn-bttm-md" data-wb-randomize='{ "shuffle": true, "selector": "a", "toggle": "hide" }'>
-	<a href="#" class="hide">
-		<figure>
-			<img class="img-responsive full-width" src="https://via.placeholder.com/1170x347/F5F5F5/000000.png" alt="">
-			<figcaption class="well brdr-rds-0">
-				<span class="h4 mrgn-tp-0 mrgn-bttm-0 mrgn-lft-0">Image caption</span>
-			</figcaption>
-		</figure>
-	</a>
-	<a href="#" class="hide">
-		<figure><img class="img-responsive full-width" src="https://via.placeholder.com/1170x347/26374A/FFFFFF.png" alt="">
-			<figcaption class="well brdr-rds-0">
-				<span class="h4 mrgn-tp-0 mrgn-bttm-0 mrgn-lft-0">Image caption</span>
-			</figcaption>
-		</figure>
-	</a>
-	<a href="#" class="hide">
-		<figure>
-			<img class="img-responsive full-width" src="https://via.placeholder.com/1170x347/7834BC/FFFFFF.png" alt="">
-			<figcaption class="well brdr-rds-0">
-				<span class="h4 mrgn-tp-0 mrgn-bttm-0 mrgn-lft-0">Image caption</span>
-			</figcaption>
-		</figure>
-	</a>
-	<a href="#" class="hide">
-		<figure>
-			<img class="img-responsive full-width" src="https://via.placeholder.com/1170x347/0000FF/000000.png" alt="">
-			<figcaption class="well brdr-rds-0">
-				<span class="h4 mrgn-tp-0 mrgn-bttm-0 mrgn-lft-0">Image caption</span>
-			</figcaption>
-		</figure>
-	</a>
-</div>
-<div class="clearfix">
-	<div class="pull-right">
-		{% include gc-follow-us/follow.html %}
+
+<div class="row">
+	<div class="col-xs-12 col-sm-8 col-md-9 col-lg-10">
+		<p>Find the most recent Government of Canada news content.</p>
+	</div>
+	<div class="col-xs-12 col-sm-4 col-md-3 col-lg-2">
+		<div class="pull-right">
+			<section class="followus mrgn-bttm-md">
```

Comment:
I don't see the effect of the margin top. do you see that effect now since the removal of the feature hero?

```suggestion
			<section class="followus">
```